image/docker_schema1.go

```diff
@@ -112,6 +113,17 @@ func (m *manifestSchema1) ConfigBlob() ([]byte, error) {
 	return nil, nil
 }
 
+// Config returns the image configuration as per OCI v1 image-spec
+func (m *manifestSchema1) Config() (*imgspecv1.Image, error) {
```

Comment:
Alright then, I'll do that (forget all the other comment I did about not doing this way, I'll follow your suggestion :))